src/app/pages/charging-stations/charging-station-limitation/charging-plans/charging-plans.component.html

```diff
@@ -5,24 +5,24 @@
         {{"chargers.smart_charging.charging_station_inactive" | translate }}
       </div>
     </div>
-    <div class="row" *ngIf="!chargingStation.inactive && isSmartChargingComponentActive && chargingStation.siteArea.smartCharging">
+    <div class="row" *ngIf="!chargingStation.inactive && isSmartChargingComponentActive && chargingStation.siteArea && chargingStation.siteArea.smartCharging">
```

Comment:
For (chargingStation.siteArea && chargingStation.siteArea.smartCharging), you can also use the shortcut with the question mark like this: (chargingStation.siteArea?.smartCharging)

Reply:
Update the code accordingly, thanks!

Reply:
Ah good to know. Thanks :)

Reply:
You welcome :-)